docs/changelog.md

```diff
@@ -0,0 +1,81 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
```

Comment:
Nit:
In *craft projects, there is vX.X.X - where contributors put the changelogs under and the X.X.X is finalized when the release happens. Would you think this approach would be easier?

Reply:
I added this to a improvement idea list for the team.